src/Test.UnitTests.Sarif.Driver/Test.UnitTests.Sarif.Driver.csproj

```diff
@@ -29,6 +29,12 @@
     <Copy SourceFiles="@(TestFiles)" DestinationFolder="$(OutputPath)\FunctionalTestData\%(RecursiveDir)" />
   </Target>
 
+  <ItemGroup>
+    <Content Include="Sdk\TestData\r.zip">
```

Comment:
What is r.zip?  recommend better naming of what is test file is.  i.e. if it is a zip with recursion or a specific depth that is to be tested, name the zip appropriate to its content.

Reply:
fixed